DependencyInjection/Configuration.php

```diff
@@ -71,6 +71,7 @@ public function getConfigTreeBuilder()
                                 ->arrayNode('proxies')
                                     ->prototype('scalar')->end()
                                 ->end()
+                                ->scalarNode('cache')->defaultValue('null')->end()
```

Comment:
http://symfony.com/doc/current/bundles/LiipImagineBundle/cache-resolver/aws_s3.html#additional-options

According to documentation it's for enabling cache itself, but ommiting it or setting it to false will still cache data, so will remove this one.